src/components/leaderboard/TopThreeCard/index.tsx

```diff
@@ -0,0 +1,34 @@
+import Image from 'next/image';
+import styles from './style.module.scss';
+
+interface UserCardProps {
+  position: 1 | 2 | 3;
+  rank: string;
+  name: string;
+  points: number;
+  image: string;
+}
+const positionNames = ['first', 'second', 'third'];
+
+const TopUserCard = ({ position, rank, name, points, image }: UserCardProps) => {
+  const trimName = name.length >= 25 ? `${name.substring(0, 24)}...` : name;
```

Comment:
Instead of writing this ternary statement, we could maybe create a trim name function in `utils.ts` making this easier to read and so that when we need to reuse this trim function in the future there'll be less code clutter

Reply:
Besides this and the changes faris talked about the code looked well structured and clean! Great job 🎉